### PR TITLE
Parse numbers into Ruby number objects.

### PR DESCRIPTION
**Pull request by dorner**

### CODE REVIEW

File: spec/grpc_rest_spec.rb [hunk 1]
```diff
@@ -66,6 +66,7 @@ def test_4(req)
     it 'should be successful' do
       params = {
         test_id: 'abc',
+        some_int: 65,
```
Comment (etam-pro) [inline spec/grpc_rest_spec.rb]: In this case if value is a string "65" instead, would it still work?

Comment (dorner) [reply spec/grpc_rest_spec.rb]: lol, no it doesn't. :facepalm: this is why everyone needs reviews!

Comment (dorner) [reply spec/grpc_rest_spec.rb]: Pushed the fix!

**Review by Conor-Flipp — APPROVED** [review]
LGTM

**Review by etam-pro — APPROVED** [review]
LGTM! Do we need to bump version?

**Pull request merged**